site/content/docs/latest/tutorials/kubeapps-on-tkg/step-3.md

```diff
@@ -5,60 +5,65 @@ Once Kubeapps is up and running on the VMware Tanzu™ Kubernetes Grid™ cluste
+- `Name` field is a friendly display name for the repository.
+- `URL` field specifies the endpoint of the repository. This endpoint might require different forms of authentication, such as `None`, `Basic Auth` (username and password), `Bearer Token` (a token) or another `Custom` mechanism.
+- `Description` field (optional) is a brief summary for the repository.
+- `Packaging format` field specifies the format of the packages available in the repository.
+- `Scope` field specifies where the packages from the repository could be installed in the cluster (globally on any namespace or limited to an specific namespace in the cluster).
+- `Package Storage Type` field specifies the type of repository. Currently, Kubeapps supports both Helm repositories and OCI registries.
 
   ![Package repositories modal part 1](../../img/kubeapps-on-tkg/apprepository-detail-general.png)
 
-- The _Repository Type_ field specifies the type of repository. Currently, Kubeapps supports both Helm repositories and OCI registries. For the latter, it is necessary to also manually specify the list of artifacts to fetch in the _List of Repositories_ field (as there is no index yet). Additionally, artifacts can be excluded using regular expressions if required.
+- `Repository Authorization` field if selected `Docker Registry` options, needs to specifies an `imagePullSecret` for images that are to be pulled from the private registry. It is possible to create a fresh secret or choose an existing one.
 
-  ![Package repositories modal part 2](../../img/kubeapps-on-tkg/apprepository-detail-types.png)
+  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-authentication.png)
 
-- The _Associate Docker Registry_ field specifies an `imagePullSecret` for images that are to be pulled from the private registry. It is possible to create a fresh secret or choose an existing one.
+- For OCI registrie, it is necessary to also manually specify the list of artifacts to fetch in the **List of Repositories** field (as there is no index yet). Additionally, artifacts can be excluded using regular expressions if required.
 
-  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-detail-credentials.png)
+  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-filtering.png)
 
-- The _Custom CA Certificate_ field specifies a CA certificate to use (with an option to skip the TLS verification if required).
-- The _Custom Sync Job_ field specifies the synchronization template to use when periodically pulling the latest changes from the application repository.
+- `Synchronization Interval` field specifies the synchronization template to use when periodically pulling the latest changes from the application repository (available for Helm via Flux and Carvel).
```

Comment:
> (available for Helm via Flux and Carvel)

There are plenty of conditional elements, like the repo filters (only available for Helm), description (just Helm), etc... perhaps we should add this message to every field... or even skip mentioning it for consistency (and for the sake of the maintainability 😅 )

Reply:
Thanks for the revision. Updated!